airtable.gemspec

```diff
@@ -1,28 +1,29 @@
-# coding: utf-8
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'airtable/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "airtable"
-  spec.version       = Airtable::VERSION
-  spec.authors       = ["Nathan Esquenazi", "Alexander Sorokin"]
-  spec.email         = ["nesquena@gmail.com", "syrnick@gmail.com"]
-  spec.summary       = %q{Easily connect to airtable data using ruby}
-  spec.description   = %q{Easily connect to airtable data using ruby with access to all of the airtable features.}
-  spec.homepage      = "https://github.com/nesquena/airtable-ruby"
-  spec.license       = "MIT"
+  spec.name        = 'airtable'
+  spec.version     = Airtable::VERSION
+  spec.authors     = ['Nathan Esquenazi', 'Alexander Sorokin']
+  spec.email       = ['nesquena@gmail.com', 'syrnick@gmail.com']
```

Comment:
Feel free to put your name here. Nathan wrote original version, so we'll put his name into Readme/Acknowledgements.

Reply:
@syrnick thx!. Will do it.

Reply:
Done